src/hotspot/share/gc/g1/g1FullCollector.hpp

```diff
@@ -95,10 +97,12 @@ class G1FullCollector : StackObj {
   G1FullGCCompactionPoint* serial_compaction_point() { return &_serial_compaction_point; }
   G1CMBitMap*              mark_bitmap();
   ReferenceProcessor*      reference_processor();
+  size_t                   hr_live_words(uint hr_index) { return _live_stats[hr_index]._live_words; }
```

Comment:
I think just `live_words` for the method name is fine, and `region_index` is the commonly used name for region indexes if you look at other code.
(Please also fix elsewhere; and in the last patch some `hr_index` also crept in into `g1RegionMarkStatsCache.cpp`).